Projects/Server/Collections/PooledRefList.cs

```diff
@@ -55,8 +55,8 @@ public PooledRefList(int capacity, bool mt = false)
         _items = capacity switch
         {
             < 0 => throw new ArgumentOutOfRangeException(nameof(capacity), capacity, CollectionThrowStrings.ArgumentOutOfRange_NeedNonNegNum),
-            0   => Array.Empty<T>(),
-            _   => (_mt ? ArrayPool<T>.Shared : STArrayPool<T>.Shared).Rent(capacity)
+            0 => Array.Empty<T>(),
+            _ => (_mt ? ArrayPool<T>.Shared : STArrayPool<T>.Shared).Rent(capacity)
```

Comment:
This is some IDE thing, and I am so annoyed. I like the alignment (I think Rider does that by default), but it isn't a standard. :|